pages/instructorAssessment/instructorAssessment.sql

```diff
@@ -238,13 +239,12 @@ FROM
     JOIN assessment_sets AS aset ON (aset.id = a.assessment_set_id)
     JOIN assessment_instances AS ai ON (ai.assessment_id = a.id)
     JOIN users AS u ON (u.user_id = ai.user_id)
-    LEFT JOIN enrollments AS e ON (e.user_id = u.user_id AND e.course_instance_id = a.course_instance_id)
+    JOIN enrollments AS e ON (e.user_id = u.user_id AND e.course_instance_id = a.course_instance_id)
```

Comment:
Sorry, this was an accident. I changed a bunch of stuff to do left joins when it wasn't necessary, and when reverting that change, I must have changed this as well.